### ISSUE TITLE
XRefStm key (Table 19) description is confusing / wrong

### ISSUE THREAD

**Issue by petervwyatt** [pdf-association/pdf-issues#146]
Adobe PDF 1.7, Table 3.17: "**XRefStm** integer (_Optional_) The byte offset from the beginning of the file of a cross-reference stream"

ISO 32000-1:2008 and ISO 32000-1:2017, Table 19: "**XRefStm** integer (_Optional_) The byte offset in the decoded stream from the beginning of the file of a cross-reference stream."

ISO ISO 32000-2:2020, Table 19: "XRefStm integer (_Optional_) The byte offset in the decoded stream from the beginning of the PDF file of a cross-reference stream."

The **XRefStm** key is in a trailer dict so I'm unclear what "... in the decoded stream" is referring to. That is clearly wrong.

And I also find the phrasing "from the beginning of the PDF file of a cross-reference stream" confusing: "of a" cross reference stream?? Surely "to the" cross reference stream or "to a" cross reference stream? 

So then there is the question of which cross reference stream to help resolve "to the" vs "to a" as this might be ambiguous... 
Clearly it is the last cross reference stream that is associated with the hybrid-reference file’s trailer dictionary and there can only be one of those so we can say "the". 

So I think it should be described as: 
 "**XRefStm** integer (_Optional_) The byte offset from the beginning of the PDF file to the cross-reference stream."

**Comment by MatthiasValvekens** [comment]
I agree the current text is confusing. I would add that it may not be immediately obvious to some readers that the "of" in "of a cross-reference stream" applies to "the byte offset" and not to "the PDF file".

However, I would contend that the indefinite article in "of a cross-reference stream" is appropriate: it's possible to have hybrid-reference files with multiple incremental updates, in which case there can be multiple trailers with **XRefStm** entries pointing to different cross-reference streams. The order in which those are to be processed is not ambiguous (or at least it isn't supposed to be), so there's no issue there IMO. The text that defines _how_ to process **XRefStm** is pretty confusing, though, and could probably do with some revising as well. Should I create a separate issue for that, or do we want to address it here?

As for the description in the table, how about this wording?

> The byte offset to a cross-reference stream, calculated from the beginning of the PDF file.

**Comment by petervwyatt** [comment]
Re: processing of **XRefStms** - please create a new issue. It will undoubtedly be a bigger issue!

I like your wording, but I still think it is to "the cross reference stream" as there can only be one directly linked from the hybrid-reference trailer dict.

So:
> The byte offset to the cross-reference stream, calculated from the beginning of the PDF file.

**Comment by petervwyatt** [comment]
PDF TWG agree
